provisioning/miniprov/hare_mp/main.py

```diff
@@ -92,7 +92,7 @@ def setup_logging(url) -> None:
 
     create_logger_directory(log_dir)
 
-    console = logging.StreamHandler(stream=sys.stderr)
+    console = logging.StreamHandler(stream=sys.stdout)
```

Comment:
We ideally need a function that takes stream as a param, all the error logs must goto stderr while regular logs to stdout. Now if we are using logging to log error messages too, then they would not goto stderr but stdout.

Reply:
@mssawant ,
Then can we do that using a different jira later and for now we can switch back to original PR change where just the "logging.info" was replaced with "logging.debug", for this urgent bug?
Anyway those mkfs prints are not critical now.

Reply:
Yeah, that should be fine.

Reply:
Done @mssawant

Reply:
I have created JIRA https://jts.seagate.com/browse/EOS-26779 for this required improvement in setup logging function.
cc: @mssawant , @knekrasov